src/VisualStudio/CSharp/Impl/CSharpPackage.cs

```diff
@@ -87,12 +94,27 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
                 });
 
                 await RegisterObjectBrowserLibraryManagerAsync(cancellationToken).ConfigureAwait(true);
+
+                CreateCodeCleanupFixers();
             }
             catch (Exception e) when (FatalError.ReportUnlessCanceled(e))
             {
             }
         }
 
+        private void CreateCodeCleanupFixers()
+        {
+            var codeCleanupFixerProvider = ComponentModel.GetService<ICodeCleanUpFixerProvider>() as CodeCleanupFixerProvider;
+            var contentTypeRegistryService = ComponentModel.GetService<IContentTypeRegistryService>();
+            var contentType = contentTypeRegistryService.GetContentType(ContentTypeNames.CSharpContentType);
+            Debug.Assert(contentType != null);
+
+            var codeFixService = ComponentModel.GetService<ICodeFixService>();
+            var threadingContext = ComponentModel.GetService<IThreadingContext>();
+            var fixer = new CSharpCodeCleanupFixer(codeFixService, threadingContext);
+            codeCleanupFixerProvider.AddFixer(contentType, fixer);
```

Comment:
can we made these to be automatically done through MEF and content type metadata?